.github/workflows/dapr.yml

```diff
@@ -90,6 +90,13 @@ jobs:
           args: --build-tags all_components
       - name: Run go mod tidy check diff
         run: make modtidy check-diff
+      - name: Check for retracted dependencies
```

Comment:
[Invalid workflow file: .github/workflows/dapr.yml#L94](https://github.com/dapr/dapr/actions/runs/5192812007/workflow)
You have an error in your yaml syntax on line 94

Reply:
Fixed.